packages/plugin-compiler/src/main/kotlin/io/realm/compiler/RealmSchemaLoweringExtension.kt

```diff
@@ -224,6 +144,96 @@ class RealmSchemaLoweringExtension : IrGenerationExtension {
     }
 }
 
+/**
+ * Locate the list of schema class literals. This currently support them being defined in-place as var args, or in place
+ * using some of the factory collection methods like setOf and arrayOf, but we don't follow the code if defined further
```

Comment:
arrayOf is not supported

Reply:
Yup, in the public API right now, but this method still supports it if we re-add it.